Floofbot/Modules/Tag.cs

```diff
@@ -4,215 +4,191 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Text;
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
-namespace Floofbot
+namespace Floofbot.Modules
 {
-    public partial class Utility
+    [Group("tag")]
+    [RequireContext(ContextType.Guild)]
+    [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
+    public class Tag : ModuleBase<SocketCommandContext>
     {
-        [Group("tag")]
-        [RequireContext(ContextType.Guild)]
-        [RequireUserPermission(Discord.GuildPermission.AttachFiles)]
-        public class Tag : ModuleBase<SocketCommandContext>
-        {
-            SqliteConnection dbConnection;
-            List<string> pages;
-
-
-            public Tag()
-            {
-                dbConnection = new SqliteConnection(new SqliteConnectionStringBuilder {
-                    DataSource = "botdata.db"
-                }.ToString());
-                dbConnection.Open();
-            }
+        private SqliteConnection dbConnection;
+        private static readonly Discord.Color embedColor = Color.Magenta;
```

Comment:
Optional style: constants should be in yelling snake case. Also, C# knowledge limitation, but would there be a functional difference between using `readonly` vs `const` here? It looks like we're not really taking advantage of the slight flexibility `readonly` has here.

Reply:
It's a bit of futureproofing. For the readonly vs const I'm following the last part of https://stackoverflow.com/a/56024

Reply:
As the for constant in screaming snake case, fixed